packages/use-sync-external-store/package.json

```diff
@@ -17,6 +17,15 @@
     "shim/",
     "cjs/"
   ],
+  "exports": {
+    ".": "./index.js",
+    "./with-selector": "./with-selector.js",
+    "./shim": "./shim/index.js",
+    "./shim/index.native": "./shim/index.native.js",
+    "./shim/with-selector": "./shim/with-selector.js",
+    "./src/*": "./src/*",
```

Comment:
Wouldn't this require changing the publish script? The latest version of `uSES` has no `src` folder. What are you currently importing with `use-sync-external-store/src/?`?

Reply:
It seems that the react package has this inline.
https://github.com/facebook/react/blob/3613284dceac38de4ef5a7c93c4897600ee6923b/packages/react/package.json#L30